site/content/en/docs/manual/advanced/delete_frame.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Frame deleting'
+linkTitle: 'Delete frame from task'
+weight: 18
+description: 'This section explains how delete and restore frame from task.'
+---
+
+# Delete frame
+
+You can delete current frame from task. This frame will not be presented either in the UI or in the exported annotation.
+Thus, it is possible to mark corrupted frames that are not subject to annotation.
+
+1. Go to the Job annotation view and than click on the `Delete frame` button.
+
+   ![](/images/image245.jpg)
+
+1. After that you will be asked to confirm frame deleting.
+   **Important note:**  all annotations from that frame will be deleted, unsaved annotations
+   will be saved and the frame will be invisible in the annotation view (Until you make it visible in the settings).
```

Comment:
@ActiveChooN , "unsaved annotations will be saved". Could you please explain? Why do we need that?

Reply:
Unsaved annotations related to the deleting frame can be there. So, as we clear all annotations from the deleted frame, it's better to save annotations before.

Reply:
@ActiveChooN , I don't understand why we need to save annotations in the case to be honest. For me it isn't consistent with current behaviors.

Reply:
Because deleting a frame also deletes all annotations on that frame. So, there can be an unsaved annotation on the frame and we need to save it first. I will try to investigate to reduce saving if there are no annotations on the frame.